.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,31 @@
+# rubi-py PR
+
+- Note - **please make sure the title of your PR follows semantic versioning** 
+- Ex. fix/feat/refactor: description 
+- For a breaking change, that will bump an entire version, please use !. Ex. refactor!: description 
```

Comment:
not verbose, exactly what is needed. the one thing here for me, i'm a bit unclear what version `refractor` will bump. i used the guide below as reference, since `refractor` wasn't there i'm still a bit unclear. i assume it acts as a feature, unless given the `!` tag?